Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11-buster as base
+
+RUN apt-get update
+RUN apt-get install -y curl
```

Comment:
This step failed when I ran it interestingly - I know we saw this working and it looked like a transient network error but the rest of the image still worked when I took it out so it looks like curl is likely already installed in your `buster` base image anyway 🙂 I'd personally leave this as is on the assumption it will resolve itself shortly, but if you do see issues, be aware you can likely resolve just by removing!